assets/crd/0000_04_routes-crd.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Where in the OpenShift repos does this CRD come from?

Reply:
https://github.com/openshift/router/blob/master/deploy/route_crd.yaml